content/en/docs/setup/cri.md

```diff
@@ -223,36 +220,61 @@ EOF
 sysctl --system
 ```
 
+### Install containerd
+
 {{< tabs name="tab-cri-containerd-installation" >}}
-{{< tab name="Ubuntu 16.04+" codelang="bash" >}}
-apt-get install -y libseccomp2
-{{< /tab >}}
-{{< tab name="CentOS/RHEL 7.4+" codelang="bash" >}}
-yum install -y libseccomp
-{{< /tab >}}
-{{< /tabs >}}
+{{< tab name="Ubuntu 16.04" codelang="bash" >}}
+# Install containerd
+## Set up the repository
+### Install packages to allow apt to use a repository over HTTPS
+apt-get update && apt-get install -y apt-transport-https ca-certificates curl software-properties-common
 
-### Install containerd
+### Add Docker’s official GPG key
+curl -fsSL https://download.docker.com/linux/ubuntu/gpg | apt-key add -
 
-[Containerd releases](https://github.com/containerd/containerd/releases) are published regularly, the values below are hardcoded to the latest version available at the time of writing. Please check for newer versions and hashes [here](https://storage.googleapis.com/cri-containerd-release).
+### Add Docker apt repository.
+add-apt-repository \
+    "deb [arch=amd64] https://download.docker.com/linux/ubuntu \
+    $(lsb_release -cs) \
+    stable"
 
-```shell
-# Export required environment variables.
-export CONTAINERD_VERSION="1.1.2"
-export CONTAINERD_SHA256="d4ed54891e90a5d1a45e3e96464e2e8a4770cd380c21285ef5c9895c40549218"
+## Install containerd
+apt-get update && apt-get install -y containerd.io
 
-# Download containerd tar.
-wget https://storage.googleapis.com/cri-containerd-release/cri-containerd-${CONTAINERD_VERSION}.linux-amd64.tar.gz
+# Configure containerd
+mkdir -p /etc/containerd
+containerd config default > /etc/containerd/config.toml
 
-# Check hash.
-echo "${CONTAINERD_SHA256} cri-containerd-${CONTAINERD_VERSION}.linux-amd64.tar.gz" | sha256sum --check -
+# Restart containerd
+systemctl restart containerd
+{{< /tab >}}
+{{< tab name="CentOS/RHEL 7.4+" codelang="bash" >}}
+# Install containerd
+## Set up the repository
+### Install required packages
+yum install yum-utils device-mapper-persistent-data lvm2
 
-# Unpack.
-tar --no-overwrite-dir -C / -xzf cri-containerd-${CONTAINERD_VERSION}.linux-amd64.tar.gz
+### Add docker repository
+yum-config-manager \
+    --add-repo \
+    https://download.docker.com/linux/centos/docker-ce.repo
 
-# Start containerd.
-systemctl start containerd
-```
+## Install containerd
+yum update && yum install containerd.io
+
+# Configure containerd
+mkdir -p /etc/containerd
+containerd config default > /etc/containerd/config.toml
+
+# Restart containerd
+systemctl restart containerd
+{{< /tab >}}
+{{< /tabs >}}
+
+To use the `systemd` cgroup driver, set `plugins.cri.systemd_cgroup = true` in `/etc/containerd/config.toml`.
```

Comment:
Suggest putting this... and the below "as well" instruction in a systemd section.

Reply:
Done!